corehq/warehouse/tests/test_staging.py

```diff
@@ -79,6 +82,7 @@ class TestGroupStagingTable(BaseStagingTableTest, StagingRecordsTestsMixin):
 
     @classmethod
     def setUpClass(cls):
+        cls.using = 'warehouse' if settings.USE_PARTITIONED_DATABASE else 'default'
```

Comment:
might be better if this referenced `WAREHOUSE_DATABASE_ALIAS` instead of `'warhouse'`

this would also be a good thing to add to the BaseWarehouseTestCase